tests/shell/shell_recorder/tutorial_wrapper/enum.md

```diff
@@ -0,0 +1,93 @@
+- infos = Information about the enum plugin is in keys below
```

Comment:
i will, that was just for demo purpose. after everything is done, i'll update them.